specification/databox/resource-manager/Microsoft.DataBox/preview/2021-08-01-preview/examples/JobMitigate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "resourceGroupName": "SdkRg9836",
+    "jobName": "SdkJob8367",
+    "api-version": "2021-03-01",
```

Comment:
It seems the API version across examples are not consistent, could you double check all the examples?